docs/user/traffic-analytics.rst

```diff
@@ -0,0 +1,43 @@
+Traffic Analytics
+=================
+
+Read the Docs aggregates statistics about visits to your documentation.
+This is mainly information about how often pages are viewed,
+and which return a `404 Not Found` error code.
+
+Traffic Analytics lets you see *which* documents your users are reading.
+This allows you to understand how your documentation is being used,
+so you can focus on expanding and updating parts people are reading most.
+
+To see a list of the top pages from the last month,
+go to the :guilabel:`Admin` tab of your project,
+and then click on :guilabel:`Traffic Analytics`.
+
+.. figure:: /_static/images/addons-analytics.png
+   :width: 50%
+   :align: center
+   :alt: Traffic analytics demo
+
+   Traffic analytics demo
+
+Enabling Traffic analytics
+--------------------------
+
+#. Go to the new :term:`dashboard`:
+#. Click on a project name.
+#. Go to :guilabel:`Settings`, then in the left bar, go to :guilabel:`Addons`.
+#. Go to :guilabel:`Analytics` and check :guilabel:`Analytics enabled`.
+
+Data storage
+------------
+
+The amount of analytics data stored for download depends which site you're using:
+
+* On |org_brand|, the last 90 days are stored.
+* On |com_brand|, it goes from 30 to infinite storage
```

Comment:
Same edits as above here -- clarifying language here and using ``seealso`` block instead.